src/CheckoutSdk/Marketplace/Company.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Checkout.Common;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Checkout.Marketplace
+{
+    public class Company
+    {
+        [JsonProperty(PropertyName = "business_registration_number")]
+        public string BusinessRegistrationNumber { get; set; }
+
+        [JsonProperty(PropertyName = "legal_name")]
+        public string LegalName { get; set; }
+
+        [JsonProperty(PropertyName = "trading_name")]
+        public string TradingName { get; set; }
+
+        [JsonProperty(PropertyName = "principal_address")]
+        public Address PrincipalAddress { get; set; }
+
+        [JsonProperty(PropertyName = "registered_address")]
+        public Address RegisteredAddress { get; set; }
```

Comment:
Please remove any `[JsonProperty]` that are not needed. Just to be clear, the serializer automatically converts from CamelCase to Snake Case (AbcDef to abc_def). Just need to add these when there are exceptions.